### ISSUE TITLE
[Visual style/colors] Provide guidance for color and accessibility

### ISSUE THREAD

**Issue by AccessForAll** [18F/govt-wide-patternlibrary#162]
* Please specify that foreground and background color contrast—for text and backgrounds/graphics—should be a minimum of 4.5:1, and preferably higher (7:1) for maximum accessibility with mobile devices, especially for those of us who practice “mobile first” methodology.

* Are we going to expect agencies to use only these colors? Some people may need to use colors that are similar, but not exactly, to meet their agency/department's contrast requirements.

* Color references:
     * Contrast (Minimum): http://www.w3.org/TR/UNDERSTANDING-WCAG20/visual-audio-contrast-contrast.html
     * Contrast (Enhanced): http://www.w3.org/TR/UNDERSTANDING-WCAG20/visual-audio-contrast7.html
     * Use of Color: http://www.w3.org/TR/UNDERSTANDING-WCAG20/visual-audio-contrast-without-color.html

**Comment by mollieru** [comment]
@AccessForAll thanks so much! We did include color recommendations intentionally in this first iteration of the patterns so that we could more deeply research and test accessible palettes. We will absolutely be including more colors and they will meet AA standards at the minimum. Thank you for the resources!
